tests/foreman/rhai/test_rhai.py

```diff
@@ -138,24 +136,7 @@ def test_positive_unregister_client_from_rhai(self):
 
                 with Session(self) as session:
                     session.nav.go_to_select_org(self.org_name)
-                    Navigator(self.browser).go_to_insights_systems()
-                    # Click on the unregister icon 'X' in the table against the
-                    # registered system listed.
-                    strategy, value = locators['insights.unregister_system']
-                    session.nav.click(
-                        (strategy, value % vm.hostname),
-                        wait_for_ajax=True,
-                        ajax_timeout=40,
-                    )
-
-                    # Confirm selection for clicking on 'Yes' to unregister the
-                    # system
-                    session.nav.click(
-                        locators['insights.unregister_button']
-                    )
-                    self.browser.refresh()
-                    time.sleep(60)
```

Comment:
So there was a necessity to wait for a minute for action to finish. Did something change in application? Are you sure vm will be updated immediately after ui action with no delays?

Reply:
I did not saw any delay in that

Reply:
The tests were very stable